docs/guides/ha-cluster/fence_nnf.py

```diff
@@ -0,0 +1,217 @@
+#!@PYTHON@
+
+# Fence agent for Near Node Flash devices
+
+import sys
+import logging
+import atexit
+
+sys.path.append("@FENCEAGENTSLIBDIR@")
```

Comment:
I think we need a comment here. Does the user need to change this?

Reply:
Sorry, should have made it more obvious in the PR. This is just a place to stuff my changes so I have a copy if my machine were to crash. The final place will be in the ClusterLabs repo (or a fork), in which case configure/make will handle the string substitution.